cc.retroshare.retroshare-gui.yaml

```diff
@@ -10,6 +10,7 @@ finish-args:
   - --socket=x11
   - --share=network
   - --persist=.retroshare
```

Comment:
You would remove this too since it no longer does anything.

Reply:
Only the persist, right? Ok, removing it.